<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HYDNBK136589" name="Test" tests="3" failures="1" timestamp="2 Apr 2018 09:24:06 GMT" time="123.204" errors="0">
  <testcase name="validateSummaryOfFundObjective" time="18.952" classname="internationalTests.Luxembourg.FOVW001_Overview_FundContent">
    <failure type="java.lang.RuntimeException" message="FAILED">
      <![CDATA[java.lang.RuntimeException: FAILED
	at utils.Reporter.reportStep(Reporter.java:35)
	at utils.Reporter.reportStep(Reporter.java:45)
	at wdMethods.ProjectMethods.printUnexpectedLogDuringExecution(ProjectMethods.java:138)
	at internationalTests.Luxembourg.FOVW001_Overview_FundContent.validateSummaryOfFundObjective(FOVW001_Overview_FundContent.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- validateSummaryOfFundObjective -->
  <testcase name="validateFundInfo" time="61.913" classname="internationalTests.Luxembourg.FOVW003_Overview_Fund_Information"/>
  <testcase name="validateNAVPrice" time="107.59" classname="internationalTests.Luxembourg.FOVW001_Overview_NAVPrice"/>
</testsuite> <!-- Test -->
